cli/functionary/package.py

```diff
@@ -17,15 +17,16 @@ def create_languages() -> list[str]:
 
 
 def generateYaml(output_dir: str, name: str, language: str):
-    metadata = {
-        "name": name,
-        "version": "1.0",
-        "x-language": language,
-    }
-
-    path = pathlib.Path(output_dir).resolve() / name / f"{name}.yaml"
-    with path.open(mode="w"):
-        path.write_text(yaml.dump(metadata))
+    path = pathlib.Path(output_dir).resolve() / name / "package.yaml"
+    path2 = pathlib.Path(
```

Comment:
As written, this will fail if you run the command anywhere outside of a very specific directory.  This should be able to be run from anywhere.  Look at how the `create` command finds the language templates.  You should be able to do the same thing.

On the topic of the language templates, that same folder is probably a more appropriate place for the package.yaml template.  So instead of having a `template.yaml` at the root, I would instead move it to `templates/package.yaml`.